yarn.lock

```diff
@@ -31167,7 +31167,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"knex@npm:^2.0.0, knex@npm:^2.3.0, knex@npm:^2.4.2":
+"knex@npm:^2.3.0":
```

Comment:
A bit suprised about this usage of `knex `^2` version. Is it still used somewhere/ a peer dependency?

Reply:
Yep that's right, the one remaining offender here is the `connect-session-knex` library mentioned above, which indeed is having type issues after upgrading the repo as we pass in knex3 clients to that knex2 library.

Reply:
Ok - let's not blocking for me than. :shipit:.

Reply:
Now it's gone! The makers of that library released a new version that builds with knex3, so I bumped that too